### PR TITLE
[2.0] FAQ: update fees for coinjoin

### PR DESCRIPTION
**Pull request by MarnixCroes**
credits @BTCparadigm https://github.com/zkSNACKs/WasabiDoc/pull/1125#pullrequestreview-1057274879

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -787,21 +787,12 @@ So there you have it! Here is the trade-off with CJ output denominations and why
...
-There are also edge cases where you do not pay the full coordinator fee or where you pay more.
-For example if you're the smallest registrant to a round, you will never pay a coordinator fee.
-Also when you are remixing and you cannot pay the full coordinator fee with your input, then you only pay as much as you have.
-But if your input is larger than the minimum, and the change amount leftover would be too small, then that is also added to the coordinator fee.
-Currently the minimum change amount to be paid out is 0.3% of the base denomination (~0.1BTC.)
-
-It is also possible that you get more back from mixing than you put in.
-This happens when network fees go down between the start of the round and its end.
-In this case, the difference is split between the active outputs of the mix.
+Coinjoining UTXOs with a value above 0.01 BTC costs 0.3% as a [coordinator](https://github.com/zkSNACKs/WalletWasabi/tree/master/WalletWasabi.Backend) fee + [mining fees](https://mempool.space/).
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:792]: I don't see a reason to link to those:

```suggestion
Coinjoining UTXOs with a value above 0.01 BTC costs 0.3% as a coordinator fee + mining fees.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -787,21 +787,12 @@ So there you have it! Here is the trade-off with CJ output denominations and why
...
-But if your input is larger than the minimum, and the change amount leftover would be too small, then that is also added to the coordinator fee.
-Currently the minimum change amount to be paid out is 0.3% of the base denomination (~0.1BTC.)
-
-It is also possible that you get more back from mixing than you put in.
-This happens when network fees go down between the start of the round and its end.
-In this case, the difference is split between the active outputs of the mix.
+Coinjoining UTXOs with a value above 0.01 BTC costs 0.3% as a coordinator fee + mining fees.
+Inputs of 0.01 BTC or below don't pay coordinator fees.
+And remixes, even after one transaction, don't pay coordinator fees. 
+Thus, a payment made with coinjoined funds allows the sender and the recipient to remix their coins without paying any coordinator fees.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:795]: How about this tweak:

```suggestion
Coinjoin fees are composed of coordination fees and mining (network) fees.
Coins with a value above 0.01 BTC pay 0.3% as a coordination fee + mining fees.
Coins of 0.01 BTC or below don't pay coordination fees.
Remixes, even after one transaction, also don't pay coordination fees. 
Thus, a payment made with coinjoined funds allows the sender and the recipient to remix their coins without paying any coordination fees.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md (hunk 2 above)
Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md:795]: good.
imo `UTXO` is better fit in this context instead of `coins`?

Comment (yahiheb) [reply docs/FAQ/FAQ-UseWasabi.md:795]: I think coins is easier to understand for the average person.
I have updated the suggestion to say `Coins (UTXOs)`.

**Review by yahiheb — APPROVED** [review]

**Pull request merged**